### PR TITLE
Issue fixes and other minor improvements

### PR DESCRIPTION
**Pull request by MaggiR**

### CODE REVIEW

File: pyofahrt/urls.py [hunk 1]
```diff
@@ -26,5 +27,5 @@
     url(r'^tasks/', include('tasks.urls', namespace='tasks', app_name='tasks')),
     url(r'^workshops/', include('workshops.urls', namespace='workshops', app_name='workshops')),
     url(r'^wiki/', include('wiki.urls', namespace='wiki', app_name='wiki')),
-    url(r'^admin/', admin.site.urls),
+    url(r'^admin/', admin.site.urls)
```
Comment (Miterion) [inline pyofahrt/urls.py]: It is python standard to add a comma at the end of a list

File: pyofahrt/urls.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 """
 from django.conf.urls import url, include
 from django.contrib import admin
+from django.contrib.auth import views as auth_views
```
Comment (Miterion) [inline pyofahrt/urls.py]: Why add this import?

File: members/static/members/roomassignment.css [hunk 3]
```diff
@@ -1,47 +1,51 @@
+#board {
```
Comment (Miterion) [inline members/static/members/roomassignment.css]: Are these changes necessary for the roomate_preference change or can they be put in an extra commit?

File: staff/models.py [hunk 4]
```diff
@@ -16,7 +16,7 @@ class Meta:
     email = models.EmailField("E-Mail-Adresse")
     phone = models.CharField("Handynummer", max_length=30)
 
-    roommate_preference = models.CharField("Gewünschte Zimmernachbarn", max_length=255, null=True)
+    roommate_preference = models.CharField("Gewünschte Zimmernachbarn", max_length=255, blank=True)
```
Comment (Miterion) [inline staff/models.py]: Use a foreign key here, for easier parsing

File: pyofahrt/urls.py (hunk 2 above)
Comment (MaggiR) [reply pyofahrt/urls.py]: I used it for some tests but I indeed forgot to remove it.

File: pyofahrt/urls.py (hunk 1 above)
Comment (MaggiR) [reply pyofahrt/urls.py]: OK, didn't know that before.

File: members/static/members/roomassignment.css (hunk 3 above)
Comment (MaggiR) [reply members/static/members/roomassignment.css]: Well, this commit contains both, reverted roommate_preference from member and improved room assignment design. But you are correct, it would be better to separate it into two commits.
